src/DynamoCoreWpf/ViewModels/Watch3D/DynamoLineGeometryModel3D.cs

```diff
@@ -41,9 +41,17 @@ protected override RenderCore OnCreateRenderCore()
             return new DynamoPointLineRenderCore();
         }
 
+#if NET6_0_OR_GREATER
+        protected override IRenderTechnique OnCreateRenderTechnique(IEffectsManager effectsManager)
+        {
+            return effectsManager[DynamoEffectsManager.DynamoMeshShaderName];
+        }
+
+#elif NETFRAMEWORK
         protected override IRenderTechnique OnCreateRenderTechnique(IRenderHost host)
         {
-            return host.EffectsManager[DynamoEffectsManager.DynamoLineShaderName];
+            return host.EffectsManager[DynamoEffectsManager.DynamoMeshShaderName];
```

Comment:
@marbelarrietaGlobant this is now the incorrect shader - this should be:
`DynamoLineShaderName `

Reply:
Fixed